etl/updated_records_capture.py

```diff
@@ -0,0 +1,107 @@
+from sqlalchemy import create_engine, text
+from datetime import datetime, timedelta
+from io import BytesIO
+import pandas as pd
+import os
+import boto3
+import io
+
+
+def date_substraction(day1, day2):
+    current_date = datetime.now().date()
+    n_1_date = str(current_date - timedelta(days=day1)).replace("-", "")
+    n_2_date = str(current_date - timedelta(days=day2)).replace("-", "")
+    return n_1_date, n_2_date
+
+
+def down_file(n_1_date, n_2_date):
+    bucket_name = "analytics-ninjas"
+    csv_buffer1 = BytesIO()
+    csv_buffer2 = BytesIO()
+    file_dir_1 = f"stock_db/stock/partition={n_1_date}/stock.csv"
+    file_dir_2 = f"stock_db/stock/partition={n_2_date}/stock.csv"
+    s3 = boto3.client(
+        "s3",
+        aws_access_key_id=os.environ["ACCESS_KEY"],
+        aws_secret_access_key=os.environ["SECRET_KEY"],
+    )
```

Comment:
No need for os.environ['ACCESS_KEY'], os.environ["SECRET_KEY"]
Already configure on VM